docs/docs/api/Tasks/getTasks.md

```diff
@@ -7,7 +7,7 @@ borderRadius: '2px',
 color: '#E83E8C',
 }}>{children}</span> );
 
-<Endpoint>GET /tasks</Endpoint>: Get all tasks <br></br>
+<Endpoint>GET /tasks?start=yyyymmdd&end=yyyymmdd</Endpoint>: Get all tasks <br></br>
```

Comment:
The get tasks page shouldn't change since this route wasn't modified. A new page should've been created for get a task within a date range (It shouldn't be in the get all tasks page)

Reply:
Please add back the original getTasks page (with any changes as needed) and make a new page for the get task within date range

Reply:
Update: The alternative would be to keep this page but put a description when the start and end dates aren't included. (i.e. If you put the start and end dates, it should filter out the tasks within that range. If omitted, then it will return all the user's tasks).

I would also rename the file to Fetch Tasks instead of Get All Tasks (If you take this approach)

Reply:
Fixed: Changes are reflected in recent commits